test/fallback.spec.ts

```
@@ -0,0 +1,26 @@
+import unrsResolver = require('unrs-resolver')
+
+import fallback = require('napi-postinstall/fallback')
+
+describe('fallback', () => {
+  afterEach(() => {
+    delete process.env.SKIP_UNRS_RESOLVER_FALLBACK
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)